docs/targets/solana.rst

```diff
@@ -23,60 +23,96 @@ This is how to build your Solidity for Solana:
 
   solang compile --target solana flipper.sol -v
 
-This will produce two files called `flipper.abi` and `flipper.so`. The former is an ethereum style abi file and the latter is
+This will produce two files called `flipper.json` and `flipper.so`. The former is an Anchor style abi file and the latter is
 the ELF BPF shared object which can be deployed on Solana. For each contract, Solang will create an ABI file and a binary file
-`contract-name.so`, which contains the code.
+`contract-name.so`, which contains the program.
+
+The program will need to be deployed. Usually, the program is deployed at a well-known address which is specified in the Solidity
+source code using the `@program_id("F1ipperKF9EfD821ZbbYjS319LXYiBmjhzkkf5a26rC")` annotation on the contract.
+In order to deploy to this address, you'll need its the private key. For the flipper example, we
+have included the private key in `flipper-keypair.json`, so you can use that here for deploying the program:
 
 .. code-block:: bash
 
-    npm install @solana/solidity
+    solana program deploy --program-id flipper-keypair.json flipper.so
```

Comment:
```suggestion
    echo "[4,10,246,143,43,1,234,17,159,249,41,16,230,9,198,162,107,221,233,124,34,15,16,57,205,53,237,217,149,17,229,195,3,150,242,90,91,222,117,26,196,224,214,105,82,62,237,137,92,67,213,23,14,206,230,155,43,36,85,254,247,11,226,145]" > flipper-keypair.json
    solana program deploy --program-id flipper-keypair.json flipper.so
```

Reply:
I think this nicely side-steps the issue of storing a private key in a repo. This also makes it easier because then the private key is always in the same directory as `flipper.so`, which makes it easier to find.